src/electos/ballotmaker/validate_edf.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+from electos.ballotmaker.constants import NO_DATA, NO_ERRORS, NO_FILE
+from electos.ballotmaker.read_edf import read_edf
+
+
+def validate_edf(
+    _edf: Path,
+) -> int:
+    """Validate EDF data
+    Requires:
+        EDF file (JSON format) edf_file: Path,
+    """
+    # is the EDF a file?
+    if _edf is None:
+        return NO_FILE
+    if Path.is_file(_edf) is False:
```

Comment:
```suggestion
    if not _edf.is_file():
```
- `is_file` is an instance method, not a class method. (What you did works here but it would go wrong if inheritance was involved.)
- Per PEP-8, don't use `is False` on boolean tests.

Reply:
Agreed and done.